utils/api_helpers.py

```diff
@@ -1,22 +1,48 @@
 import aiohttp
-from groq import AsyncGroq
+from groq import AsyncGroq as ai
 
 import config
+import logger
 from utils.data import Fursona
 
-LANGUAGE_MODEL = "llama-3.3-70b-versatile"
-VISION_MODEL = "llama-3.2-11b-vision-preview"
+log = logger.get_logger(__name__)
```

Comment:
Remove the logger because I don't think it's needed anymore with my proposed changes